services/src/main/webapp/WEB-INF/web.xml

```diff
@@ -58,6 +58,10 @@
             <web-resource-name>Public</web-resource-name>
             <url-pattern>/ArquillianServletRunner/*</url-pattern>
         </web-resource-collection>
+        <web-resource-collection>
```

Comment:
I had a lot of issues with that one, but I think that I have found a simpler approach. I'm testing it now.